rust/operator-binary/src/controller.rs

```diff
@@ -10,21 +10,24 @@ use crate::{
 
 use indoc::formatdoc;
 use snafu::{OptionExt, ResultExt, Snafu};
+use stackable_operator::builder::PodSecurityContextBuilder;
+use stackable_operator::cluster_resources::ClusterResourceApplyStrategy;
+use stackable_operator::commons::rbac::build_rbac_resources;
```

Comment:
Could be moved down into `stackable_operator`.

Reply:
Corrected.